mods/ts/rules/shared-vehicles.yaml

```diff
@@ -130,6 +130,59 @@ LPST:
 			gdi: lpst.gdi
 			nod: lpst.nod
 
+LPST2:
+	Inherits: ^Vehicle
+	Valued:
+		Cost: 950
+	Tooltip:
+		Name: Awesomer Mobile Sensor Array
+	Buildable:
+		Queue: Vehicle
+		BuildPaletteOrder: 100
+		Prerequisites: ~factory, radar
+	Health:
+		HP: 600
+	Armor:
+		Type: Light
+	Mobile:
+		Speed: 85
+		ROT: 5
+	RevealsShroud:
+		Range: 10c0
+	-AppearsOnRadar:	# TODO: Discuss whether we want to remove the trait. The original had RadarInvisible=yes
```

Comment:
Let's spare the balancing discussion for now and just keep original Tiberian Sun behavior while we are still working on the technical parts.